### PR TITLE
Update the apim.jwt_authenitcation configurations

### PR DESCRIPTION
**Pull request by sanethmaduranga**
## Purpose
This will add the configurations for the subscription validation from the KM.

## Documentation
https://apim.docs.wso2.com/en/latest/learn/api-security/oauth2/access-token-types/jwt-tokens/#secure-apis-using-jwt-self-contained-access-tokens

### CODE REVIEW

File: en/docs/learn/api-security/oauth2/access-token-types/jwt-tokens.md [hunk 1]
```diff
@@ -247,3 +247,7 @@ If you can't satisfy the above requirements but still need subscription validati
 
 1. Navigate to the `deployment.toml` under the `<API-M_HOME>/repository/conf/` folder.
 2. Add the **subscription_validation_via_km** configuration under the `[apim.jwt_authenitcation]`.
+  ```
+[apim.jwt_authenitcation]
+subscription_validation_via_km = true
```
Comment (shaniR) [inline en/docs/learn/api-security/oauth2/access-token-types/jwt-tokens.md:252]: @ruthryi  this should go to the config catalog?

Comment (shaniR) [reply en/docs/learn/api-security/oauth2/access-token-types/jwt-tokens.md:252]: @tharindu1st  can you please review this?

Comment (tharindu1st) [reply en/docs/learn/api-security/oauth2/access-token-types/jwt-tokens.md:252]: this was correct

**Comment by shaniR** [comment]
@sanethmaduranga please send this change to the master as well as include it into the config catalog.

**Pull request merged**